FileManipulator.Test/FileManipulator.Test.csproj

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk; Microsoft.Build.CentralPackageVersions">
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <RootNamespace>Messerli.FileManipulator.Test</RootNamespace>
+        <LangVersion>8.0</LangVersion>
+        <Nullable>enable</Nullable>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <WarningsAsErrors />
```

Comment:
Is this needed?

Reply:
Already addressed

Reply:
https://github.com/messerli-informatik-ag/meta-generator/pull/14/commits/6c407ef640eae4e4ada1e956055861c135345bde